File: jscript/j20.arguments.js
WScript.Echo("The arguemnts passed are (reverse order):");

// print count and current argument
for(var count=WScript.Arguments.length; count > 0; count--) { 
  WScript.Echo(" item " + (count) + ": " + WScript.Arguments.Item(count-1)); 
}

File: lessons/compiled_lang/rust/src/j20.arguments.rs
use std::env;

fn main() {
    let args: Vec<String> = env::args().skip(1).collect();

    println!("The arguments passed are (reverse order):");
    for i in (0..args.len()).rev() {
        println!(" item {}: {}", i + 1, args[i]);
    }
}
